mlrun/utils/logger.py

```diff
@@ -352,11 +356,20 @@ def bind(self, **kw_args):
     def _update_bound_vars_and_log(
         self, level, message, *args, exc_info=None, **kw_args
     ):
+        request_id = request_id_var.get() or "N/A"
+
         kw_args.update(self._bound_variables)
 
         if kw_args:
             self._logger.log(
-                level, message, *args, exc_info=exc_info, extra={"with": kw_args}
+                level,
+                message,
+                *args,
+                exc_info=exc_info,
+                extra={
+                    "with": kw_args,
+                    "request_id": request_id,
```

Comment:
```suggestion
                    "ctx": request_id,
```

request_id is just a usage of a context-based information, meaning, request id is usually for api request id. sometimes you have internal procedure running, and its context is not really a request_id.
`ctx` makes it consistent with other services and a bit more generic.